positions.go

```diff
@@ -46,6 +46,18 @@ type LegPosition struct {
 	OptionType     string `json:"option_type"`
 }
 
+type CryptoPosition struct {
+	Meta
+	Id             			string  `json:"id"`
```

Comment:
missed alignment

Reply:
@daniel-baez I can't believe you've done this